src/App.js

```diff
@@ -10,13 +10,25 @@ class App extends Component {
     super();
     this.state = {
       movies: movieData.movies,
-      individualMovie: null
+      individualMovie: null,
+      error: null
     }
   }
 
+  componentDidMount = () => {
+    fetch('https://rancid-tomatillos.herokuapp.com/api/v2/movies')
+			.then((response) => response.json())
+			.then((data) => {this.setState({movies: data.movies});
+			})
+			.catch((error) => {
+        this.setState({ error:`Sorry looks like there is a ${error.message} error, please try again later`});
```

Comment:
I hope this part works! I wonder how we test for 500 errors when we aren't running the server ?

Reply:
Nice work adding that error handling for the `.catch()`.  Unfortunately there's not a great way to test 500 errors since the server should stay up and running on Heroku.  You could probably stub one though in your Cypress tests!